src/PrestaShopBundle/Resources/config/services/core/form/form_builder.yml

```diff
@@ -226,3 +226,11 @@ services:
       arguments:
           - 'PrestaShopBundle\Form\Admin\Configure\ShopParameters\SearchEngines\SearchEngineType'
           - '@prestashop.core.form.identifiable_object.data_provider.search_engine_form_data_provider'
+
+# @todo: its actually not identifiable object. Do we care?
```

Comment:
I don't think it's that important, the main interest here is to get the `actionCategoryTreeSelectorFormBuilderModifier` hook

Reply:
You can remove the todo comment